app/metrics.yaml

```diff
@@ -3195,6 +3195,66 @@ top_sites:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-11-15"
+  open_frecency:
+    type: event
+    description: |
+      A user opened a frecency top site
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/14565
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/TODO
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-12-15"
```

Comment:
Could you move these to 3/15 since that will be 6mo from now?

Reply:
Done!